### PR TITLE
DDF-5676 fixed Solr build on Windows

### PR DESCRIPTION
**Pull request by stustison**
#### What does this PR do?
Fixes Solr build on Windows
#### Who is reviewing it? 

@bakejeyner @rzwiefel 

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/solr-distro/src/main/resources/server/solr/zoo.cfg [hunk 1]
```diff
@@ -14,4 +14,4 @@ syncLimit=5
 # clientPort=2181
 
 # override default buffer size to 20k
```
Comment (pklinef) [inline distribution/solr-distro/src/main/resources/server/solr/zoo.cfg]: :pencil2: Update the comment from `20k` to `2MB` or `2097151 bytes`

**Comment by stustison** [comment]
build now

**Comment by bakejeyner** [comment]
✋ Heroing

**Comment by stustison** [comment]
build now

**Pull request closed without merging**